lib/Myriad/Exception/RPC.pm

```diff
@@ -0,0 +1,29 @@
+package Myriad::Exception::RPC;
+
+# VERSION
+# AUTHORITY
+
+use Myriad::Exception::Builder;
+
+declare_exception InvalidRequest => (
+    package => 'Myriad::Exception::RPC',
```

Comment:
This shouldn't be needed - if it is, you could extract it out to the `use Myriad::Exception::Builder` line as a default